test/unit/any_instance_method_test.rb

```diff
@@ -6,9 +6,16 @@
 class AnyInstanceMethodTest < Mocha::TestCase
   include Mocha
 
+  def class_with_method(method, result = nil)
+    Class.new do
+      extend ClassMethods
+      define_method(method) { result } if method
+    end
+  end
+
```

Comment:
I'd prefer that the `class_with_method` method was initially extracted in a separate preparatory commit. I think that would make it clearer what was actually changing in this commit.

Reply:
Resolving - the whole branch has been re-written.